ignite/metrics/mutual_information.py

```diff
@@ -0,0 +1,110 @@
+from typing import Sequence
+
+import torch
+import torch.nn.functional as F
+
+from ignite.exceptions import NotComputableError
+from ignite.metrics.metric import Metric, reinit__is_reduced, sync_all_reduce
+
+__all__ = ["MutualInformation"]
+
+
+class MutualInformation(Metric):
+    r"""Calculates the `mutual information <https://en.wikipedia.org/wiki/Mutual_information>`_
+    between input :math:`X` and prediction :math:`Y`.
+
+    .. math::
+
+       I(X;Y) = H(Y) - H(Y|X)
+       = H \left( \frac{1}{N}\sum_{i=1}^N \hat{\mathbf{p}}_i \right)
+       - \frac{1}{N}\sum_{i=1}^N H(\hat{\mathbf{p}}_i),
```

Comment:
Let's add new-line tag here. Otherwise it is rendered as a single line. See here: https://deploy-preview-3230--pytorch-ignite-preview.netlify.app/generated/ignite.metrics.mutualinformation#ignite.metrics.MutualInformation